armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -168,6 +169,8 @@ cat << EOF > /root/.bashrc-custom
 export LS_OPTIONS='--color=auto'
 alias l='ls $LS_OPTIONS -l'
 alias ll='ls $LS_OPTIONS -la'
+alias bbbconfig='bbb-config.sh'
```

Comment:
(Optional) Personally I probably wouldn't add these aliases, since they're not that much shorter than the name of the scripts they are aliasing anyway, and it seems like having more than one way to do the same thing could lead to more confusion.